apps-rendering/src/item.ts

```diff
@@ -367,150 +374,151 @@ const isPicture = hasTag('type/picture');
-		if (isInteractive(content)) {
-			return {
-				design: ArticleDesign.Interactive,
-				...itemFieldsWithBody(context, request),
-			};
-			// This isn't accurate, picture pieces look different to galleries.
-			// This is to prevent accidentally breaking Editions until we have
-			// a model for pictures.
-		} else if (isGallery(tags) || isPicture(tags)) {
-			return {
-				design: ArticleDesign.Gallery,
-				...itemFieldsWithBody(context, request),
-			};
-		} else if (isAudio(tags)) {
-			return {
-				design: ArticleDesign.Audio,
-				...itemFieldsWithBody(context, request),
-			};
-		} else if (isVideo(tags)) {
-			return {
-				design: ArticleDesign.Video,
-				...itemFieldsWithBody(context, request),
-			};
-		} else if (isReview(tags)) {
-			return {
-				design: ArticleDesign.Review,
-				starRating: fromNullable(fields?.starRating),
-				...itemFieldsWithBody(context, request),
-			};
-		} else if (isAnalysis(tags)) {
-			return {
-				design: ArticleDesign.Analysis,
-				...itemFieldsWithBody(context, request),
-			};
-		} else if (isCorrection(tags)) {
-			return {
-				design: ArticleDesign.Correction,
-				...itemFieldsWithBody(context, request),
-			};
-		} else if (isLetter(tags)) {
-			return {
-				design: ArticleDesign.Letter,
-				...itemFieldsWithBody(context, request),
-			};
-		} else if (isObituary(tags)) {
-			return {
-				design: ArticleDesign.Obituary,
-				...itemFieldsWithBody(context, request),
-			};
-		} else if (isGuardianView(tags)) {
-			return {
-				design: ArticleDesign.Editorial,
-				...itemFieldsWithBody(context, request),
-			};
-		} else if (isComment(tags)) {
-			const item = itemFieldsWithBody(context, request);
-			return {
-				design: ArticleDesign.Comment,
-				...item,
-				theme:
-					item.theme === ArticlePillar.News
-						? ArticlePillar.Opinion
-						: item.theme,
-			};
-		} else if (isInterview(tags)) {
-			return {
-				design: ArticleDesign.Interview,
-				...itemFieldsWithBody(context, request),
-			};
-		} else if (isFeature(tags)) {
-			return {
-				design: ArticleDesign.Feature,
-				...itemFieldsWithBody(context, request),
-			};
-		} else if (isLive(tags)) {
-			return fromCapiLiveBlog(context)(request, page);
-		} else if (isRecipe(tags)) {
-			return {
-				design: ArticleDesign.Recipe,
-				...itemFieldsWithBody(context, request),
-			};
-		} else if (isQuiz(tags)) {
-			return {
-				design: ArticleDesign.Quiz,
-				...itemFieldsWithBody(context, request),
-			};
-		} else if (isLabs(tags)) {
 			return {
 				design: ArticleDesign.Standard,
 				...itemFieldsWithBody(context, request),
-				theme: ArticleSpecial.Labs,
 			};
-		} else if (isMatchReport(tags)) {
-			return {
-				design: ArticleDesign.MatchReport,
-				football: parseMatchScores(
-					fromNullable(request.footballContent),
-				),
-				...itemFieldsWithBody(context, request),
-			};
-		}
-
-		return {
-			design: ArticleDesign.Standard,
-			...itemFieldsWithBody(context, request),
 		};
-	};
```

Comment:
Are there code changes here, or is this all indentation? If this and the above change to `fromCapiLiveBlog` are both just indentation would it be ok if you reverted them? I think it might make this PR easier to review and may cause less confusion for anyone digging through the blame for this file in future.

Reply:
Just whitespace changes. I've reverted these and the ones further up so it doesn't muddy the water!